sfn-query-large-datasets-cdk/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.swp
+package-lock.json
```

Comment:
Being picky here but I assume this `package-lock.json` is not really needed as this is python project.

Reply:
done.